src/solver/model_getters.jl

```diff
@@ -225,6 +225,8 @@ function get_radiation_mode(parsed_args, ::Type{FT}) where {FT}
     @assert add_isothermal_boundary_layer in (true, false)
     aerosol_radiation = parsed_args["aerosol_radiation"]
     @assert aerosol_radiation in (true, false)
+    reset_rng_seed = parsed_args["radiation_reset_rng_seed"]
+    @assert reset_rng_seed in (true, false)
```

Comment:
Can you please add a warning when this option is turned on for radiation models that are not RRTGMP and allsky? Users should be notified that something that they explicitely asked is incompatible with their setup.